python-checks/src/main/resources/org/sonar/l10n/py/rules/python/S5843.html

```diff
@@ -3,43 +3,6 @@
 <p>The complexity of a regular expression is determined as follows:</p>
 <p>Each of the following operators increases the complexity by an amount equal to the current nesting level and also increases the current nesting
 level by one for its arguments:</p>
-<ul>
```

Comment:
is this change expected?

Reply:
This piece of metadata seems related to the Java implementation of the rule and duplicate of the Python one starting line 43 .